src/App.scss

```diff
@@ -17,13 +17,3 @@ iframe {
   padding: 20px;
```

Comment:
You combined here 4 different styles, sidebar, page and iframe. Each class should be related to his component. Split these styles to different styles or rename them according to your component